HW_5/Task_1.py

```diff
@@ -0,0 +1,15 @@
+# Напишите функцию, которая принимает на вход строку — абсолютный путь до файла.
+# Функция возвращает кортеж из трёх элементов: путь, имя файла, расширение файла.
+
+text = r'C:\Users\79282\PycharmProjects\PYTHON_HW\Seminar_1\Task_1.py'
+
+
+def file_path(text):
+    *path, name = text.split('\\')
+    name = list(name.split('.'))
```

Comment:
Зачем функция `list,`  Метод `split` уже возвращает этот тип данных.
```suggestion
    name = name.split('.')
```